docker-compose.yml

```diff
@@ -139,3 +150,4 @@ services:
       - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN
     depends_on:
       - influxdb
+    attach: false
```

Comment:
I´m getting an error here: 
_services.dqDatabase Additional property attach is not allowed_
The error message seems to rotate between the services, when I retry to compose up.

Reply:
https://docs.docker.com/engine/reference/commandline/compose_up/ It is a real property.. it allows to not show logs of specific container when running `docker compose up`